### PR TITLE
capitalized the doctype in the example html

### PR DESCRIPTION
**Pull request by telnettrauma**
### Description

Capitalized the DOCTYPE in an example HTML document.

### Motivation

More consistent with the rest of the document

### Additional details

https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode

### CODE REVIEW

File: files/en-us/web/html/quirks_mode_and_standards_mode/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The limited-quirks and no-quirks modes used to be called "almost-standards" mode
 For [HTML](/en-US/docs/Web/HTML) documents, browsers use a DOCTYPE in the beginning of the document to decide whether to handle it in quirks mode or standards mode. To ensure that your page uses full standards mode, make sure that your page has a DOCTYPE like in this example:
 
 ```html
-<!doctype html>
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (sideshowbarker) [reply files/en-us/web/html/quirks_mode_and_standards_mode/index.md:20]: MDN style for HTML snippets is to lowercase it — as the linter suggestion indicates

**Comment by Josh-Cena** [comment]
@sideshowbarker I think we should lowercase the other "Doctype" references on this page, which are not in code blocks and therefore not autoformatted. The HTML spec also calls it "Doctype" instead of "DOCTYPE". WDYT? Would you want to send a PR and I could review it?

**Comment by sideshowbarker** [comment]
> @sideshowbarker I think we should lowercase the other "Doctype" references on this page, which are not in code blocks and therefore not autoformatted. The HTML spec also calls it "Doctype" instead of "DOCTYPE". WDYT? Would you want to send a PR and I could review it?

I somebody else did a PR for it, I’d similarly by happy to review it

**Comment by Josh-Cena** [comment]
=> https://github.com/mdn/content/pull/35707

**Pull request closed without merging**